runtime/ftplugin/man.vim

```diff
@@ -20,11 +20,12 @@ setlocal wrap breakindent linebreak
 setlocal nonumber norelativenumber
 setlocal foldcolumn=0 colorcolumn=0 nolist nofoldenable
 
+setlocal tagfunc=man#goto_tag
+
 if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
   nnoremap <silent> <buffer> j          gj
   nnoremap <silent> <buffer> k          gk
   nnoremap <silent> <buffer> gO         :call man#show_toc()<CR>
-  nnoremap <silent> <buffer> <C-]>      :Man<CR>
   nnoremap <silent> <buffer> K          :Man<CR>
   nnoremap <silent> <buffer> <C-T>      :call man#pop_tag()<CR>
```

Comment:
should be able to remove `<C-T>` too, I guess.

Reply:
ah yeah, cheers :+1: